snapshots/CaliburExecuteTest.json

```diff
@@ -1,7 +1,7 @@
 {
   "execute_BATCHED_CALL_SUPPORTS_OPDATA_singleCall": "95804",
   "execute_BATCHED_CALL_SUPPORTS_OPDATA_twoCalls": "132454",
-  "execute_BATCHED_CALL_opData_P256_singleCall": "121112",
+  "execute_BATCHED_CALL_opData_P256_singleCall": "121088",
```

Comment:
why is this test changing but not the other ones? I thought that it could be because of the change in the 712 domain name but that should affect all signature ones?

Reply:
calldata cost changes bc of the changing domain separator